doc/data/messages/m/magic-value-comparison/details.rst

```diff
@@ -0,0 +1 @@
+You can help us make the doc better `by contributing <https://github.com/PyCQA/pylint/issues/5953>`_ !
```

Comment:
I think there's still a problem here following resolution of conflict for the merge of main I just did. This new file should not exists. Would you mind rebasing on main and cleaning up the doc generation commit by doing a partial add of only the broad-except-x related changes ? Sorry for the mess, we did not have time to fix #6736 yet or this would not be a problem.

Reply:
done, I think the latest looks most accurate, let's see how CI reacts